Clock.ino

```diff
@@ -214,7 +222,10 @@ HourHand hourHand_;
 void setup()
 {
     initClockPins();
-    rtcClock_.init(globTime_);    
+    rtcClock_.init(globTime_); 
+
+    // send all axes to zero, then tell objects to reset to zero
+    zero_all(100, 1000);   
```

Comment:
this should be handled separately in the classes

Reply:
how should we do it at the same time within the classes? should there be a motor class so we can have motor objects and then you can zero all of the motors and then instantiate a gantry with the x and y motor object?

Reply:
The Motor class would work in steps but the gantry or hour hand could work in millimeters or degrees

Reply:
I made the motor class and separated the zeroing of the gantry and hour hand